nsis/README.txt

```diff
@@ -16,10 +16,10 @@ To build the installable .exe:
 	install.exe,
 	uninstall.exe,
 	tee/tee.exe,
-	xxd/xxd.exe,
+	xxd/xxd.exe
 
-    Then execute tools/rename.bat to rename the executables. (mv command is
-    required.)
+    Then execute tools/rename.bat to rename the executables (on Unix-like,
+    it must first be made executable; chmod +x path/to/vimsrc/tools/rename.bat).
```

Comment:
How about setting the executable flag in this repository beforehand?

Reply:
Oh! That would be great! Could you tell me how to do it? It's just that my main tool is Windows.
I'm going to try to log in to the Ubuntu via VM and set this flag in my fork. Will that be enough?

Reply:
Changing the flag on a unix-like system should be the easiest way and that should be enough.
However, on Windows, you can use `git update-index --add --chmod=+x <filename>`.